js_intro_and_control_flow.js

```diff
@@ -17,73 +17,58 @@ var friends = [
                 'Harriet'
               ];
 
-Add your name to the end of the `friends` and add another name to beginning. Change the `Elizabeth` to `Liz`.
+//Add your name to the end of the `friends` and add another name to beginning. Change the `Elizabeth` to `Liz`.
 
+var lastName = friends[(friends.length-1)];
 
+friends.push("Jamie");
 
+friends.unshift("Tom");
 
+friends.splice(6, 1, "Liz");
```

Comment:
Cool. And if you didn't know the contents of the array you could use `friends.indexOf('Elizabeth')` to find the index.